common/services/document.js

```diff
@@ -14,6 +14,17 @@ const logErrorMessage = error => {
 }
 
 const documentService = {
+  create(data) {
+    return Promise.resolve({
+      filename: data.originalname,
+      id: data.filename,
+    })
+  },
+
+  destroy(id) {
+    return Promise.resolve(id)
+  },
+
   async upload(file, moveId) {
```

Comment:
Does anything delete the files out of the temporary directory once they are uploaded? Also thinking that ./tmp/ will be a path inside the container and wondering how much space we will have?

Reply:
Currently, no. But I'm working on another branch to address how the files are stored. And looking to create a separate storage class for uploading the files to the API instead of saving to disk.